components/pixel/schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "title": "Pixel Component Definition",
+  "description": "Addressable Pixel WipperSnapper component for use in Adafruit IO",
+  "type": "object",
+  "required": [ "displayName", "pixelsType", "defaultPixelsOrder"],
+  "properties": {
+    "displayName": {
+      "description": "The human-friendly name of this component.",
+      "type": "string",
+      "minLength": 3,
+      "maxLength": 24
+    },
+    "pixelsType": {
+        "description": "The type of addressable pixel: NEOPIXEL or DOTSTAR. Corresponds to `PixelsType` in pixels.proto",
+        "type": "string",
+        "pattern": "^(NEOPIXEL|DOTSTAR)$"
+    },
+    "defaultPixelsOrder": {
+        "description": "The pixel strand's color ordering. Corresponds to `defaultPixelsOrder` in pixels.proto.",
+        "type": "string",
+        "pattern": "^(GRB|GRBW|RGB|RGBW|BRG)$"
+    },
+    "defaultNumPixels": {
+        "description": "If present, the component form will allow the user to set the number of pixels in the strand, with this value as the default.",
+        "type": "number",
+        "minimum": 0,
+        "maximum": 156
+    },
+    "defaultPixelBrightness": {
+        "description": "If present, the component form will allow the user to set the pixel strand's brightness, with this value as the default.",
```

Comment:
Same sort of questions as with num pixels.

Reply:
This is only set **once**, during initialization (see: https://github.com/adafruit/Wippersnapper_Protobuf/pull/111/files#diff-a1604995e7a859acfac92b0c1873d95f017ae9712b241d7b1713035ab5ce6636R40). There is a default brightness I can pass in internally of `255/2`. This field is required to be set in the form.